package.json

```diff
@@ -10,7 +10,6 @@
     "konva"
   ],
   "bugs": "https://github.com/konvajs/react-konva/issues",
-  "licenses": "MIT",
```

Comment:
Why did you remove the license?

Reply:
@lavrton It's already under the `licence` key, this is a duplicate `licence`**`s`** (multiple) key

Reply:
Ah, right! I didn't notice that.